pages/common/glab-auth.md

```diff
@@ -0,0 +1,20 @@
+# glab auth
+
+> Authenticate with a GitLab host from the command-line.
+> More information: <https://glab.readthedocs.io/en/latest/auth>.
+
+- Log in with interactive prompt:
+
+`glab auth login`
+
+- Log in with a token (created in https://gitlab.com/-/profile/personal_access_tokens):
```

Comment:
This could be created in an Enterprise GitLab server, I think we shouldn't include this link.
```suggestion
- Log in with a token:
```